components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -587,6 +587,27 @@ - (void)updateTopHeaderBottomShadowWithContentOffset:(CGPoint)contentOffset {
   }
 }
 
+- (void)updateContentWithHeight:(CGFloat)height {
+  if (height < 0) {
+    height = 0;
+  }
+  // This is added so we don't recursively add height
+  CGFloat previousAddedHeight = self.addedHeight;
+  self.addedHeight = height;
+  CGFloat heightToAdd = self.addedHeight - previousAddedHeight;
+  if (self.contentViewController) {
+    CGRect contentViewFrame = CGRectStandardize(self.contentViewController.view.frame);
+    contentViewFrame.size =
+        CGSizeMake(contentViewFrame.size.width, contentViewFrame.size.height + heightToAdd);
+    self.contentViewController.view.frame = contentViewFrame;
+  } else if (self.headerViewController) {
+    CGRect headerViewFrame = CGRectStandardize(self.headerViewController.view.frame);
```

Comment:
we shouldn't support cases where there is no contentViewController given, at this time. Seems like an odd behavior which wasn't tested yet and isn't part of the spec.

Reply:
Removed